x-pack/plugins/uptime/server/lib/alerts/status_check.ts

```diff
@@ -297,7 +307,7 @@ export const statusCheckAlertFactory: UptimeAlertTypeFactory<ActionGroupIds> = (
       isAutoGenerated,
       timerange: oldVersionTimeRange,
     } = rawParams;
-
+    const { basePath } = _server;
```

Comment:
```suggestion
    const { basePath } = server;
```

Reply:
`server` doesn't exist in this file. only `_server` exists https://github.com/fkanout/kibana/blob/364fb0b0d94607b07b8a966c3b34ec8fab7e3dff/x-pack/plugins/uptime/server/lib/alerts/status_check.ts#L233

Reply:
we can only also rename that :)

_ is only added when var is unused , so to not trigger lint warning.

Reply:
@shahzad31, Aha, I see, I didn't know about this naming convention. I will update it.